domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -1922,7 +1922,7 @@ public TotaledResults(RowSet<Row> set, Integer estimatedTotal) {
 
   private void closeIfNonNull(Transaction transaction) {
     if (transaction != null) {
-      transaction.close();
+      transaction.completion();
```

Comment:
completion() does not do anything, it only returns a Future.
You need to call commit().

```
private Future<Void> commitIfNonNull(Transaction transaction) {
  if (transaction == null) {
    return Future.succeededFuture();
  }
  return transaction.commit();
}
```

Reply:
done.